src/user/helper.js

```diff
@@ -10,16 +10,22 @@ export async function createUser(xo, params) {
 
 export async function deleteAllUsers(xo) {
```

Comment:
```js
export async function deleteAllUsers(xo) {
  await Promise.all(
    userIds.map(id =>
      xo.call("user.delete", { id }).catch(error => console.error(error))
    )
  );
  userIds.length = 0;
}
```

Reply:
But if the first userId is deleted correctly and the second is not, I think we must delete the first one and keep the second in userIds. 
What do you think ?